testing/version.go

```diff
@@ -20,6 +20,11 @@ func (v semver) atLeast(other semver) bool {
 			return true
 		}
 	}
+	for i := len(v); i < len(other); i++ {
+		if 0 < other[i] {
```

Comment:
```suggestion
		if other[i] > 0 {
```

I think the intention reads a little bit more clearly if we flip this condition.